### PR TITLE
[Core] Fix user output on cog load/reload

### PR DESCRIPTION
**Pull request by mikeshardmind**
### Type

- [X] Bugfix
- [X] Enhancement
- [ ] New feature

### Description of the changes
  - Properly fixes the load/reload exception handling
  - Fixes some i18n use here to not make assumptions about other
  languages pluralization rules.

### CODE REVIEW

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -584,38 +565,64 @@ class Core(commands.Cog, CoreLogic):
...
+                ).format(packs=humanize_list([inline(package) for package in failed]))
+            output.append(formed)
 
         if not_found:
-            fmt = _("The package{plural} {packs} {other} not found in any cog path.")
-            formed = self._get_package_strings(not_found, fmt, (_("was"), _("were")))
-            await ctx.send(formed)
+            if len(not_found) == 1:
+                formed = _("The following package was not found in any cog path: {pack}.").format(
+                    inline(not_found[0])
```
Comment (Flame442) [inline redbot/core/core_commands.py]: This throws a key error because `pack` is not found in the `.format()`

File: redbot/core/core_commands.py [hunk 2]
```diff
@@ -584,38 +565,64 @@ class Core(commands.Cog, CoreLogic):
...
+                )
+            else:
+                formed = _(
+                    "The following packages were not found in any cog path: {packs}"
+                ).format(humanize_list([inline(package) for package in not_found]))
```
Comment (Flame442) [inline redbot/core/core_commands.py]: This throws a key error because `packs` is not found in the `.format()`

File: redbot/core/core_commands.py [hunk 3]
```diff
@@ -648,30 +674,53 @@ class Core(commands.Cog, CoreLogic):
...
 
         if failed:
-            fmt = _("Failed to reload package{plural} {packs}. Check your logs for details")
-            formed = self._get_package_strings(failed, fmt)
-            await ctx.send(formed)
+            if len(failed) == 1:
+                formed = _(
+                    "Failed to reload the following package: {packs}."
+                    "\nCheck your console or logs for details."
+                ).format(inline(failed[0]))
```
Comment (Flame442) [inline redbot/core/core_commands.py]: This throws a key error because `packs` is not found in the `.format()`

File: redbot/core/core_commands.py [hunk 4]
```diff
@@ -648,30 +674,53 @@ class Core(commands.Cog, CoreLogic):
...
+                ).format(packs=humanize_list([inline(package) for package in failed]))
+            output.append(formed)
 
         if not_found:
-            fmt = _("The package{plural} {packs} {other} not found in any cog path.")
-            formed = self._get_package_strings(not_found, fmt, (_("was"), _("were")))
-            await ctx.send(formed)
+            if len(not_found) == 1:
+                formed = _("The following package was not found in any cog path: {pack}.").format(
+                    inline(not_found[0])
```
Comment (Flame442) [inline redbot/core/core_commands.py]: This throws a key error because `pack` is not found in the `.format()`

File: redbot/core/core_commands.py [hunk 5]
```diff
@@ -648,30 +674,53 @@ class Core(commands.Cog, CoreLogic):
...
+                )
+            else:
+                formed = _(
+                    "The following packages were not found in any cog path: {packs}"
```
Comment (Flame442) [inline redbot/core/core_commands.py:706]: This throws a key error because `packs` is not found in the `.format()`

**Review by Flame442 — CHANGES_REQUESTED** [review]

**Review by Flame442 — DISMISSED** [review]

**Review by Tobotimus — DISMISSED** [review]

**Review by Tobotimus — APPROVED** [review]

**Pull request merged**